ODKAggregateAppEngineUpdater.jar errors out with Java 9

When I try to install Aggregate on macOS running Java 9, I get this message. To reproduce, double-click the uploadAggregateToAppEngine.app app and hit getToken.

```
getToken :  Beginning interaction for module default...
getToken!:  Mar 16, 2018 2:46:03 PM com.google.appengine.tools.admin.Application guessContentTypeFromName
getToken!:  WARNING: Error identify mimetype for /api/appversion/getresourcelimits
getToken!:  java.lang.NoClassDefFoundError: javax/activation/FileTypeMap
getToken!:  	at com.google.appengine.tools.admin.Application.guessContentTypeFromName(Application.java:584)
getToken!:  	at com.google.appengine.tools.admin.AbstractServerConnection.send1(AbstractServerConnection.java:334)
getToken!:  	at com.google.appengine.tools.admin.AbstractServerConnection.send(AbstractServerConnection.java:317)
getToken!:  	at com.google.appengine.tools.admin.AbstractServerConnection.post(AbstractServerConnection.java:296)
getToken!:  	at com.google.appengine.tools.admin.NoLoggingClientDeploySender.send(NoLoggingClientDeploySender.java:35)
getToken!:  	at com.google.appengine.tools.admin.ResourceLimits.remoteRequest(ResourceLimits.java:174)
getToken!:  	at com.google.appengine.tools.admin.ResourceLimits.request(ResourceLimits.java:139)
getToken!:  	at com.google.appengine.tools.admin.AppAdminImpl.getResourceLimits(AppAdminImpl.java:328)
getToken!:  	at com.google.appengine.tools.admin.AppCfg$ResourceLimitsInfoAction.execute(AppCfg.java:1983)
getToken!:  	at com.google.appengine.tools.admin.AppCfg.executeAction(AppCfg.java:393)
getToken!:  	at com.google.appengine.tools.admin.AppCfg.<init>(AppCfg.java:216)
getToken!:  	at com.google.appengine.tools.admin.AppCfg.<init>(AppCfg.java:116)
getToken!:  	at com.google.appengine.tools.admin.AppCfg.main(AppCfg.java:112)
getToken!:  Caused by: java.lang.ClassNotFoundException: javax.activation.FileTypeMap
getToken!:  	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
getToken!:  	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
getToken!:  	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
getToken!:  	... 13 more
getToken!:  
status :  Action Succeeded!
```

The app runs `java -jar ./ODKAggregateAppEngineUpdater.jar` and when I run that command with Java 1.8, it works great.

--- a/buildSrc/src/main/groovy/org/opendatakit/aggregate/gradle/gae/Rollback.groovy
+++ b/buildSrc/src/main/groovy/org/opendatakit/aggregate/gradle/gae/Rollback.groovy
@@ -29,7 +29,7 @@ class Rollback extends JavaExec {
   }
 
   String getSDKHome() {
-    return "${getWorkingDir()}/appengine-java-sdk-1.9.54".toString()
+    return "${getWorkingDir()}/appengine-java-sdk-1.9.63".toString()
   }
 
   String getDummyAppPath() {

--- a/buildSrc/src/main/groovy/org/opendatakit/aggregate/gradle/gae/Update.groovy
+++ b/buildSrc/src/main/groovy/org/opendatakit/aggregate/gradle/gae/Update.groovy
@@ -18,7 +18,7 @@ class Update extends JavaExec {
   }
 
   String getSDKHome() {
-    return "${getWorkingDir()}/appengine-java-sdk-1.9.54".toString()
+    return "${getWorkingDir()}/appengine-java-sdk-1.9.63".toString()
   }
 
   String getEarPath() {

--- a/gae.gradle
+++ b/gae.gradle
@@ -8,9 +8,9 @@ import static org.opendatakit.aggregate.gradle.Util.truncate
 task gaeDownloadSDK() {
   doLast {
     if (!file('installer/appEngineSDK.zip').exists()) {
-      println("Downloading App Engine SDK 1.9.54")
+      println("Downloading App Engine SDK 1.9.63")
       download {
-        src 'https://storage.googleapis.com/appengine-sdks/featured/appengine-java-sdk-1.9.54.zip'
+        src 'https://storage.googleapis.com/appengine-sdks/featured/appengine-java-sdk-1.9.63.zip'
         dest 'installer/appEngineSDK.zip'
       }
     } else

--- a/installer.gradle
+++ b/installer.gradle
@@ -34,6 +34,6 @@ task installerBuild(dependsOn: [installerClean, gaeDownloadSDK], type: Copy) {
   doLast {
     setXmlValue("${buildDir}/installer/buildWar.xml", "version", version)
     file("${buildDir}/installer/files/${archivesBaseName}-${version}.war").renameTo(file("build/installer/files/ODKAggregate.war"))
-    file("${buildDir}/installer/files/appengine-java-sdk-1.9.54").renameTo(file("build/installer/files/appengine-java-sdk"))
+    file("${buildDir}/installer/files/appengine-java-sdk-1.9.63").renameTo(file("build/installer/files/appengine-java-sdk"))
   }
 }

--- a/installer.gradle
+++ b/installer.gradle
@@ -35,5 +35,15 @@ task installerBuild(dependsOn: [installerClean, gaeDownloadSDK], type: Copy) {
     setXmlValue("${buildDir}/installer/buildWar.xml", "version", version)
     file("${buildDir}/installer/files/${archivesBaseName}-${version}.war").renameTo(file("build/installer/files/ODKAggregate.war"))
     file("${buildDir}/installer/files/appengine-java-sdk-1.9.63").renameTo(file("build/installer/files/appengine-java-sdk"))
+    delete "${buildDir}/installer/files/appengine-java-sdk/demos"
+    delete "${buildDir}/installer/files/appengine-java-sdk/docs/endpoints"
+    delete "${buildDir}/installer/files/appengine-java-sdk/docs/javadoc"
+    delete "${buildDir}/installer/files/appengine-java-sdk/docs/remoteapi"
+    delete "${buildDir}/installer/files/appengine-java-sdk/docs/spi"
+    delete "${buildDir}/installer/files/appengine-java-sdk/docs/testing"
+    delete "${buildDir}/installer/files/appengine-java-sdk/docs/tools"
+    delete "${buildDir}/installer/files/appengine-java-sdk/jetty93"
+    delete "${buildDir}/installer/files/appengine-java-sdk/jetty93-base"
+    delete "${buildDir}/installer/files/appengine-java-sdk/src"
   }
 }

